EX1/HW7.py

```diff
@@ -0,0 +1,373 @@
+import math
+
+
+class Poll:
```

Comment:
Documantation :

“Code tells you how; Comments tell you why.”

— [Jeff Atwood]

https://realpython.com/documenting-python-code/#commenting-vs-documenting-code